web/yo/app/views/driveRealtimeString.html

```diff
@@ -40,19 +40,27 @@ <h4 ng-if="reviewMode && !rs.get('action')">Old Content:</h4>
     <div ng-if="t === 'treatment-select'" class="treatment-select">
         <div>
             <h4 ng-if="reviewMode && !rs.get('action')">New Content:</h4>
-            <select ng-model="content.stringO" ng-if="!reviewMode || !rs.get('action')"
-                    ng-options="opt.value as opt.label for opt in o">
-            </select>
+            <div ng-if="!reviewMode || !rs.get('action')">
```

Comment:
originally the action is undefined. we set it during the review process to control the colors and existence of accept and reject icons. Possible values are 'rejected', 'accepted'.